pydy/utils.py

```diff
@@ -33,6 +33,12 @@ def sympy_equal_to_or_newer_than(version, installed_version=None):
     return parse_version(v) >= parse_version(version)
 
 
+def sympy_newer_than(version):
+    """Returns true if the installed version of SymPy is newer than the
+    provided version string."""
+    return parse_version(SYMPY_VERSION) > parse_version(version)
```

Comment:
This can probably be removed now because SymPy no longer has the trailing "-git" on the dev version numbers.